ddtrace/contrib/internal/logging/patch.py

```diff
@@ -68,6 +68,9 @@ def _get_tracer(tracer=None):
 def _w_makeRecord(func, instance, args, kwargs):
     # Get the LogRecord instance for this log
     record = func(*args, **kwargs)
+    if not config._logs_injection:
+        # log injection is opt-in for non-structured logging
```

Comment:
Log injection for unstructured logs is only doable with custom instrumentation for now, right?

Reply:
Re:
> Using an [unstructured](https://docs.python.org/3/howto/logging-cookbook.html#implementing-structured-logging) logger is not supported.

Do you mean that we don't support **structured** logs? (Looks like this sentence is just a typo)